### PR TITLE
BXC-4686 & BXC-4687 - Add cropping report and API

### PR DESCRIPTION
**Pull request by lfarrell**
https://unclibrary.atlassian.net/browse/BXC-4686
https://unclibrary.atlassian.net/browse/BXC-4687

### CODE REVIEW

File: web-admin-app/src/main/java/edu/unc/lib/boxc/web/admin/controllers/ChompbController.java [hunk 1]
```diff
@@ -48,16 +51,45 @@ public String chompbCroppingReport() {
...
+                                                @PathVariable("jobName") String jobName,
+                                                @RequestParam(value = "path", defaultValue = "false") String filename)
+                                                throws IOException {
+//        var filename = extractFilename(request, jobName);
         var agentPrincipals = AgentPrincipalsImpl.createFromThread();
-        try {
-            return chompbPreIngestService.getProcessingResults(agentPrincipals, projectName, jobName);
-        } catch (IOException e) {
-            throw new RuntimeException(e);
+//        filename = URLDecoder.decode(filename, StandardCharsets.UTF_8);
```
Comment (lfarrell) [inline web-admin-app/src/main/java/edu/unc/lib/boxc/web/admin/controllers/ChompbController.java]: Should this be removed?

File: static/js/vue-cdr-access/src/router.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ const router = createRouter({
 });
 
 router.beforeEach((to, from) => {
+  console.log('access');
```
Comment (lfarrell) [inline static/js/vue-cdr-access/src/router.js:69]: We can remove this. I think I left this one in. :)

**Review by lfarrell — COMMENTED** [review]
A couple of stray commented out lines can be removed

File: web-admin-app/src/main/java/edu/unc/lib/boxc/web/admin/controllers/ChompbController.java (hunk 1 above)
Comment (bbpennel) [reply web-admin-app/src/main/java/edu/unc/lib/boxc/web/admin/controllers/ChompbController.java]: I think this got removed already if you look at the final state of the PR

**Review by sharonluong — APPROVED** [review]

**Pull request merged**